crates/eww/src/config/system_stats.rs

```diff
@@ -229,3 +231,17 @@ pub fn net() -> String {
 pub fn get_time() -> String {
     chrono::offset::Utc::now().timestamp().to_string()
 }
+
+pub fn get_ipv4() -> String {
+    let ifas = list_afinet_netifas().unwrap();
+    let joined =
+        ifas.iter().filter(|ipv| ipv.1.is_ipv4() && ipv.0 != "lo").map(|ip| format!("{}", ip.1)).collect::<Vec<_>>().join(", ");
+    joined
```

Comment:
```suggestion
    ifas.iter().filter(|ipv| ipv.1.is_ipv4() && ipv.0 != "lo").map(|ip| format!("{}", ip.1)).collect::<Vec<_>>().join(", ")
```
jsut return this directly